app/assets/javascripts/home.js.coffee

```diff
@@ -3,8 +3,133 @@
 # You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
 
 $ ->
+  if namespace.controller is 'home' and namespace.action is 'index'
+
+    # Determines the current Bootstrap screen size being used to resize UI elements
+    screenSize = (size) ->
+      return ($ ".device-#{size}").is(':visible')
+
+    ###
+    #Resize UI components based upon Bootstrap and screen size
+    ###
+    resizeUI = () ->
+      if screenSize('md') or screenSize('sm')
```

Comment:
Ewwwwww grosssssssss make it staph make it stahp yuck yuck yuck get it off. Instead of having tons of jqueries doing the style, we should probably be making use of bootstrap + scss -> look at media queries. Ideally, none of the styling has to be done dynamically with coffeescript. http://getbootstrap.com/css/#grid-example-fluid